src/routes/docs/configuring/ftp/+page.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Connecting to FTP servers
+---
+
+Files has built-in support for connecting to remote FTP and FTPs servers.
+This allows you to effortlessly connect to any server that supports these protocols, browse the directory structure, and perform tasks such as editing, renaming, and deleting files or folders according to your needs.
+
+## How to connect to FTP servers
+
+You can connect to remote servers from the address bar
+
+**FTP:** `ftp://<server_address>:<port>`.  
+**FTPs:** `ftps://<server_address>:<port>`.
+
+If you don't specify the port, the default port will be used:
+
+**FTP:** `21`.  
+**FTPs:** `990`.
+
+Then if dialog appears, enter your credentials or login as anonymous user.  
+After that, you will be able to view the directory structure of the server.
```

Comment:
I'm wondering if this is implied, we might be able to remove this line.

Reply:
Or maybe we can write that entering credentials (if needed) is the last step?

Reply:
I meant to remove this line "After that, you will be able to view the directory structure of the server"

Reply:
Yeah, it will be look better.